TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js

```diff
@@ -593,6 +605,8 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
 				}.observes('aspectRatioWidth', 'aspectRatioHeight').on('init'),
 
 				_aspectRatioDidChange: function () {
+					parent.set('_showOriginalImageInInspector', false);
+					parent.set('_showOriginalImageInInspector', true);
```

Comment:
I saw setting the observed property `false` and `true` again here:
https://github.com/neos/neos-development-collection/pull/1460/files#diff-c19abf5ce59d6240278511d0413a4631L398

I figured the observer works more reliable this way